compiler/src/iree/compiler/DispatchCreation/Passes.cpp

```diff
@@ -235,6 +235,15 @@ addDispatchRegionCreationPasses(OpPassManager &passManager,
                                            /*fuseMultiReduction=*/false,
                                            /*fuseTruncateOps=*/true});
       })
+      // 5. After all the reshape propagations, fuse elementwise operations
+      //    even if the producer has multiple uses.
+      .addPass([] {
+        FuseMultiUseElementwiseProducerPassOptions options;
+        options.intraDispatch = true;
```

Comment:
Why do we have the option being `false` by default, but we use `true` in the pipeline?